src/QuickCheckVEngine/RVFI_DII.hs

```diff
@@ -48,6 +48,8 @@
 module QuickCheckVEngine.RVFI_DII
   ( module QuickCheckVEngine.RVFI_DII.RVFI,
     module QuickCheckVEngine.RVFI_DII.DII,
+    RvfiDiiConnection,
```

Comment:
This just exports the type name; try `RvfiDiiConnection(..)` (I know, the syntax is... charming.)

Reply:
Fixed in https://github.com/CTSRD-CHERI/QuickCheckVEngine/pull/7/commits/082ae836ecd770a9aab74464398fc4f3a32b6270, discovered that syntax while looking at some other files.